src/views/tutorial-view/components/next-previous/helpers.ts

```diff
@@ -3,22 +3,23 @@ import {
 	CollectionLite as ClientCollectionLite,
 	TutorialLite as ClientTutorialLite,
 } from 'lib/learn-client/types'
-import {
-	getCollectionSlug,
-	getTutorialSlug,
-} from 'views/collection-view/helpers'
 import { NextPreviousProps } from '.'
 
 interface GetNextPreviousParams {
 	currentTutorialSlug: string
 	currentCollection: ClientCollection
 	nextCollectionInSidebar: ClientCollectionLite
+	formatting: {
```

Comment:
The waf tutorials and collections have a different slug structure, introducing this to handle that.